liana-gui/src/export.rs

```diff
@@ -282,6 +284,28 @@ impl Export {
             ImportExportType::ImportPsbt => import_psbt(&sender, path).await,
             ImportExportType::ImportDescriptor => import_descriptor(&sender, path).await,
             ImportExportType::ExportBackup(str) => export_string(&sender, path, str).await,
+            ImportExportType::ExportProcessBackup(datadir, network, config, wallet) => {
+                let backup = match app_backup(
+                    datadir,
+                    network,
+                    config,
+                    wallet,
+                    daemon.clone().expect("cannot fail"),
+                    &sender,
+                )
+                .await
+                {
+                    Ok(b) => {
+                        let _ = sender.send(Progress::Progress(80.0));
```

Comment:
there is already `let _ = sender.send(Progress::Progress(60.0));` at the end of the app_backup fn, I think you do not need it here